### PR TITLE
test: add test for graphql-docs-generator

### PR DESCRIPTION
**Pull request by tacck**
#### Description of changes

Increase test coverage of amplify-graphql-docs-generator.

#### Description of how you validated changes

lerna --scope amplify-graphql-docs-generator run test

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by attilah** [comment]
@tacck Please rebase your PR as we have merged a change since your PR was created to fix the Java install problem.

**Comment by tacck** [comment]
@attilah Thanks for you comment, and sorry for using old master.
I rebased my branch.

File: packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { loadSchema } from '../../../src/generator/utils/loading';
+
+describe('loading', () => {
+  it('JSON schema', () => {
+    const graphqlSchema = loadSchema(__dirname + '/../../../fixtures/schema.json');
```
Comment (cjihrig) [inline packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts:5]: Can you put the fixtures directory in a variable that can be reused across these tests.

Comment (cjihrig) [inline packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts:5]: We should probably use Node's `path` module to create these paths.

File: packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    expect(graphqlSchema.constructor.name).toBe('GraphQLSchema');
+  });
+
+  it('JSON schema without data property', () => {
+    const graphqlSchema = loadSchema(__dirname + '/../../../fixtures/schemaWithoutDataProperty.json');
+    expect(graphqlSchema.constructor.name).toBe('GraphQLSchema');
+  });
+  it('JSON schema file is not exits', () => {
```
Comment (cjihrig) [inline packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts:13]: Please add a newline between the tests. Same thing between lines 21 and 22.

Comment (cjihrig) [inline packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts:13]:
```suggestion
  it('JSON schema file does not exist', () => {
```

File: packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts (hunk 1 above)
Comment (tacck) [reply packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts]: Thanks for your review.
I fixed it.

Comment (tacck) [reply packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts]: Thanks for your review.
I fixed it.

File: packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts (hunk 2 above)
Comment (tacck) [reply packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts]: Thanks for your review.
I fixed it.

Comment (tacck) [reply packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts]: Thanks for your review.
I fixed it.

File: packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { loadSchema } from '../../../src/generator/utils/loading';
+import { join } from 'path';
+
+const FIXTURE_PATH = join(__dirname, '/../../../fixtures');
```
Comment (cjihrig) [inline packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts:4]:
```suggestion
const FIXTURE_PATH = path.resolve(__dirname, '..', '..', '..', 'fixtures');
```

**Review by cjihrig — COMMENTED** [review]
One last small nit, but LGTM after that.

**Pull request merged**